carrot-log4j-aws-sns/src/main/java/com/carrotgarden/log4j/aws/sns/Appender.java

```diff
@@ -216,14 +225,14 @@ protected boolean ensureCredentials() {
 
             if (url != null) {
                 try {
-                    amazonCredentials = new PropertiesCredentials(url.openStream());
+                    amazonCredentials = new StaticCredentialsProvider(new PropertiesCredentials(url.openStream()));
```

Comment:
Why StaticCredentialsProvider over PropertiesCredentials (for either side of this condition)? Is this a remnant from the previous implementation from today?